modules/social_features/social_user/asyncapi.yml

```diff
@@ -269,6 +269,141 @@ channels:
+                      nullable: true
+                      description: The first name of the user.
+                    lastName:
+                      type: string
+                      nullable: true
+                      description: The last name of the user.
+                    email:
+                      type: string
+                      format: email
+                      description: The email address of the user.
+                    roles:
+                      type: array
+                      items:
+                        type: string
+                      description: Roles assigned to the user.
+                    timezone:
+                      type: string
+                      description: The timezone of the user.
+                    language:
+                      type: string
+                      description: The preferred language of the user.
+                    address:
+                      type: object
+                      nullable: true
+                      properties:
+                        label:
+                          type: string
+                          description: The label of the user address.
+                        countryCode:
+                          type: string
+                          description: The country code of the user address.
+                        administrativeArea:
+                          type: string
+                          description: The administrative area of the user address.
+                        locality:
+                          type: string
+                          description: The locality of the user address.
+                        dependentLocality:
+                          type: string
+                          description: The dependent locality of the user address.
+                        postalCode:
+                          type: string
+                          description: The postal code of the user address.
+                        sortingCode:
+                          type: string
+                          description: The sorting code of the user address.
+                        addressLine1:
+                          type: string
+                          description: The first address line of the user address.
+                        addressLine2:
+                          type: string
+                          description: The second address line of the user address.
+                    phone:
+                      type: string
+                      nullable: true
+                      description: The phone number of the user.
+                    function:
+                      type: string
+                      nullable: true
+                      description: The job function of the user.
+                    organization:
+                      type: string
+                      nullable: true
+                      description: The organization of the user.
```

Comment:
```suggestion
```

As part of the payload design we left this out.

Reply:
@bramtenhove You are right. It's fixed. Thanks!